### ISSUE TITLE
Downloads of csv files

### ISSUE THREAD

**Issue by matthewjay15** [UCL-ECHILD/code.echild.ac.uk-website#42]
The links on individual pages to the Github *.csv files open as text and as *.txt files when right-clicking to download. Ideally these should download directly on left-click as *.csv files.

**Comment by tony-stone** [comment]
There's definitely a way to get a download location programmatically however I think there might be an easier solution if you can grab the download url like this:
![image](https://github.com/user-attachments/assets/67f7688b-9fa2-4e19-bb76-4143cec5227b)

**Comment by matthewjay15** [comment]
Ah thanks. I was taking the link from raw but will switch to this.

**Comment by matthewjay15** [comment]
I can't seem to gran the download URL

**Comment by tony-stone** [comment]
The links actually look good. I think we just need to indicate to browsers that the browser should download -rather than open- the linked file. See https://www.w3schools.com/tags/att_download.asp

**Comment by tony-stone** [comment]
Curly braces content for links seems to be some poorly documented quarto-specific Markdown-flavour. Will do this in plain html.

**Comment by tony-stone** [comment]
#69 reopened, attempted close by #70 - failed.

**Comment by tony-stone** [comment]
Reopened by #73

**Comment by tony-stone** [comment]
Reopened by #73

**Comment by tony-stone** [comment]
Reopened by #75

**Comment by tony-stone** [comment]
Reopened by #75

**Comment by tony-stone** [comment]
Unfortunately, the `download` attribute of links only works for same-origin URLs or the blob: and data: schemes. Ref: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes

It would be possible to copy the csv files hosted on the phenotypes-code-list repo across when rendering the website and upload these so they _are_ then hosted on the same origin. Whether this is worthwhile or not is debateable.

Despite this issue not being resolved, I think #76 has other -worthwhile- improvements so I don't propose reverting it.

**Comment by matthewjay15** [comment]
Agreed. It's probably not worth, especially given most users will (I expect) be accessing the files on the SRS, anyway. The right-click solution is fine for now, I'd say.

**Comment by matthewjay15** [comment]
This is now fixed by including the code list files and scripts within the website's repo rather than pointing to a separate repo.
